update_model.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+from memote.suite.cli.reports import diff
+import cobra
+import os
+import logging
+from cobra.manipulation.modify import rename_genes
+#from concerto.utils.biolog_help import add_biolog_exchanges
+from cobra.io import read_sbml_model
+
+log = logging.getLogger()
+
+_file_path = os.getcwd()
+starting_model_f_name = 'csc052.xml'
+s_model_path = os.path.join(_file_path, starting_model_f_name)
+
+starting_model = cobra.io.read_sbml_model(s_model_path)
+starting_model.id = "csc052"
+
+output_model_name = 'updated_csc052.xml'
+output_model_path = os.path.join(_file_path, output_model_name)
+
+model = read_sbml_model("csc009.xml")
+#model = read_sbml_model("csc031.xml")
```

Comment:
These extra fields can be removed as this model is only concerned with the uncommented `model=`.